lib/protocol/collection/notes_collection.ts

```diff
@@ -0,0 +1,94 @@
+import { SNNote, SNTag, SNSmartTag, SNPredicate, SNItem, ContentType } from '../../models'
+import { ItemCollection, CollectionSort, SortDirection } from "./item_collection";
+import { UuidString } from "@Lib/types";
+
+export class NotesCollection {
+  private displayedList: SNNote[] = [];
+  private tag?: SNTag;
+  needsRebuilding = true;
+
+  constructor(private collection: ItemCollection) {
+  }
```

Comment:
Not retaining passed in collection object?

Reply:
When you specify the modifier in the constructor it auto-assigns it: https://www.typescriptlang.org/play?#code/MYGwhgzhAECC0G8BQ1rAPYDsIBcBOArsDungBQAOeAlgG5g4Cm0ARgFzSYEC2LjeASgAFCAL5JRQA